docs/content/preview/yugabyte-cloud/cloud-basics/cloud-vpcs/cloud-add-endpoint.md

```diff
@@ -12,16 +12,16 @@ menu:
 type: docs
 ---
 
-A private service endpoint (PSE) is used to connect a YugabyteDB Managed cluster that is deployed in a VPC with other services on the same cloud provider - typically a VPC that hosts an application that you want to have access to your cluster. The PSE connects to an endpoint attached to a VPC hosting your application over a privately linked service. Unlike VPC peering, when connected to a VPC using a private link, you do not need to add an IP allow list to your cluster.
+A private service endpoint (PSE) is used to connect a YugabyteDB Managed cluster that is deployed in a VPC with other services on the same cloud provider - typically a VPC that hosts an application that you want to have access to your cluster. The PSE on your cluster connects to an endpoint on a VPC hosting your application over a privately linked service. Unlike VPC peering, when connected to a VPC using a private link, you do not need to add an IP allow list to your cluster.
```

Comment:
modify "The PSE on your cluster connects to an endpoint on a VPC hosting your application over a privately linked service" 

to "The PSE on your cluster connects to an endpoint on a VPC hosting your application over a private connection"